### ISSUE TITLE
Remove and replace usages of duplicate class

### ISSUE THREAD

**Issue by eeverman** [eeverman/andhow#384]
The class:
`org.yarnandtail.compile.IOUtil` (remove)
completely duplicates another class:
`org.yarnandtail.andhow.util.IOUtil` (keep)

These classes are in different modules, so [this pom file](https://github.com/eeverman/andhow/blob/master/andhow-testing/andhow-annotation-processor-test-harness/pom.xml) will need a reference to the the core module like this:
```
<dependency>
	<groupId>${project.groupId}</groupId>
	<artifactId>andhow-core</artifactId>
	<version>${project.version}</version>
	<scope>provided</scope>
</dependency>
```
(See [this pom file](https://github.com/eeverman/andhow/blob/master/andhow-annotation-processor/pom.xml) for an example of another place where this type of dependency is added).

Other than additional methods in the _keep_ IOUtil, these two classes are the same with the exception of the _keep_ `toString` method, which has a bug fix in it that the _remove_ IoUtil class does not have.

**Comment by Casva** [comment]
I can pick this one up!

**Comment by Casva** [comment]
Btw @eeverman I noticed that [this pom file](https://github.com/eeverman/andhow/blob/master/andhow-testing/andhow-simulated-app-tests/pom.xml) doesn't have any reference to the core module either.

Is that supposed to be the case?

**Comment by eeverman** [comment]
To answer your question @Casva - The `andhow-simulated-app-tests` module is just a logical wrapper around integration testing.  The children under that module handle their own dependencies.

If it that dependency was actually needed, the build would fail, so we can be pretty sure :-)

**Comment by rupeshA** [comment]
I want to give it a try, can you please provide the required permissions.

**Comment by eeverman** [comment]
Hi @rupeshA - Another user, @Casva is already working on this.
No special permissions are required - Anyone can put in a merge request.

**Comment by eeverman** [comment]
Fixed by #386
